src/act.wizard.cpp

```diff
@@ -2511,7 +2511,7 @@ ACMD(do_award)
   k = atoi(amt);
 
   if (!*arg || !*amt || !*reason || k <= 0 ) {
-    send_to_char("Syntax: award <player> <karma x 100> <Reason for award>.\r\n", ch);
+    send_to_char("Syntax: award <player> <karma x 100> [for] <Reason for award>\r\n", ch);
```

Comment:
Same note re: `[for]` not being removed.

Reply:
Syntax reminder for staff, does not actually affect the logging or input line.